internal/repotesting/repotesting.go

```diff
@@ -102,10 +91,6 @@ func (e *Environment) Close(ctx context.Context, t *testing.T) {
 		// should be empty, assuming Disconnect was successful
```

Comment:
The `Remove` above, although redundant, it is a safety check. The alternative would be to list the contents and check for an empty list.